File: java/src/main/java/com/iteso/nintendo/Scratch.java
package com.iteso.nintendo;
/**Ataque Scratch.*/
public class Scratch implements Attack {

    /**Nombre del ataque.*/
    private final String name = "Ataque Scratch";
    /**Daño del ataque.*/
    private final int damage = 18;

    /**@return Nombre del ataque.*/
    @Override
    public String getAttackName() {
        return this.name;
    }

    /**@return Daño del ataque.*/
    @Override
    public int getAttackDamage() {
        return this.damage;
    }
}

File: csharp/Clases/Scratch.cs
namespace unittestpractice.Clases
{
    public class Scratch : IAttack
    {

        /**Nombre del ataque.*/
        private const string Name = "Ataque Scratch";
        /**Daño del ataque.*/
        private const int Damage = 17;

        public string GetAttackName()
        {
            return Name;
        }

        public int GetAttackDamage()
        {
            return Damage;
        }
    }
}